krun.py

```diff
@@ -265,7 +265,11 @@ def inner_main(mailer, on_first_invocation, config, args):
             util.fatal(error_msg)
 
         debug("Using pre-recorded initial temperature readings")
-        platform.starting_temperatures = current.starting_temperatures
+        manifest = ManifestManager(config, platform)
+        platform.starting_temperatures = {
+            sensor: tup[1] for sensor, tup in
+            manifest.starting_temperatures.iteritems()
```

Comment:
I flattened that to a loop then.

[Note the dict comprehension syntax is almost identical to list comprehension syntax: `[x for x in y]` vs. `{k: v for k, v in y}`. Maybe it's just that this was a non-trivial comprehension that it looked odd]